src/core/ext/transport/binder/transport/binder_transport.cc

```diff
@@ -595,8 +608,14 @@ static void perform_transport_op_locked(void* transport_op,
     gbt->state_tracker.RemoveWatcher(op->stop_connectivity_watch);
   }
   if (op->set_accept_stream) {
-    gbt->accept_stream_fn = op->set_accept_stream_fn;
     gbt->accept_stream_user_data = op->set_accept_stream_user_data;
+    gbt->accept_stream_fn = op->set_accept_stream_fn;
```

Comment:
I think we will also want to add a `GPR_INFO`/`GPR_DEBUG` log here (log would be helpful even when `gbt->accept_stream_fn_called_count_` is `0`)